libPhoneNumber-Demo/libPhoneNumber-Demo/GeocodingTableView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  GeocodingTableView.swift
+//  libPhoneNumber-Demo
+//
+//  Created by Rastaar Haghi on 7/17/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+import SwiftUI
+import libPhoneNumber_iOS
+import libPhoneNumberGeocoding
+
+struct GeocodingTableView: View {
+    // Keep track of runtime statistics
+    var maxRuntime: CGFloat = 0.00
+    var minRuntime: CGFloat = 0.00
+    var totalRuntime: CGFloat = 0.00
+    var averageRuntime: CGFloat = 0.00
+    
+    init() {
+        for _ in 1...50 {
```

Comment:
500 instead of 50?

Reply:
each call to the function actually makes 10 api calls (iterates through array of phone numbers), so iterating to 50 actually makes 500 api calls